Cargo.toml

```diff
@@ -11,10 +11,11 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+bdk_wallet = { version = "1.0.0-beta.1" }
```

Comment:
Should this be an optional dependency?

Reply:
I think it can become optional if we make `signer` a feature.

I don't see any problem in having it as a feature, I think it's a good idea.

Reply:
I'll put it behind a `signer` feature, and set `bdk_wallet` as optional.